microsoft-edge/toc.yml

```diff
@@ -73,52 +76,6 @@
 
```

Comment:
Delete the copy of these lines that are at the same indent as "- name: Microsoft Edge 100":
```
            - name: Microsoft Edge 99
              href: devtools-guide-chromium/whats-new/2022/03/devtools.md
            - name: Microsoft Edge 98
              href: devtools-guide-chromium/whats-new/2022/02/devtools.md
            - name: Microsoft Edge 97
              href: devtools-guide-chromium/whats-new/2022/01/devtools.md
```

Reply:
fixed